userspace/libsinsp/CMakeLists.txt

```diff
@@ -199,6 +200,10 @@ if(USE_BUNDLED_JSONCPP)
 	add_dependencies(sinsp jsoncpp)
 endif()
 
+if(USE_BUNDLED_BS_THREADPOOL)
+	add_dependencies(sinsp bs_threadpool)
```

Comment:
Note: if you accept on not setting a thread pool by default in an inspector (keeping it null by default), this dependency will not be necessary anymore. `bs_threadpool` would become a dependency just for the sinsp unit tests. `threa_pool_bt.cpp` would be included just if the module is used. This way adopters can discard the new dependency if they don't need it.